java/src/jmri/jmrit/display/layoutEditor/LayoutBlockConnectivityTools.java

```diff
@@ -436,7 +436,8 @@ public boolean checkValidDest(LayoutBlock facing, LayoutBlock protecting, Facing
             }
             return returnBlocks;
         }
-        LayoutBlock currentLBlock = protectingLayoutBlock;
+        //TODO: FindBugs says this is a dead store; code-strip it
+        LayoutBlock currentLBlock; // = protectingLayoutBlock;
```

Comment:
I think that FindBugs was warning you this the assignment at this line was never used, not that this variable is not used. (I see that currentLBlock is written to at line 456, and used in lines 460, 463, and 470). You could move the declaration of currentLBlock to line 456 if that makes where this value is used clearer.

Reply:
Done.